cent/client/client.py

```diff
@@ -0,0 +1,115 @@
+from typing import List, Optional, Any, Dict, TypeVar
+
+from cent.client.session import BaseSyncSession, RequestsSession
+from cent.methods import (
+    CentMethod,
+    BroadcastMethod,
+    PublishMethod,
+    SubscribeMethod,
+)
+from cent.types import (
+    StreamPosition,
+    PublishObject,
```

Comment:
Let's call this PublishResult to follow naming in api.proto. And for other similar types